### DISCUSSION TITLE
Home Assistant Assistent kennt/erkennt z.B. keine Fenster

### DISCUSSION THREAD

**Discussion by itn3rd77** [SukramJ/aiohomematic#986] [Q&A]
Hallo zusammen,

ich habe mal ein bischen mit dem Assistenten von Home Assistant gespielt, aber bekomme bei den meisten Anfragen nur "unzureichende" Antworten ;-)

![grafik](https://user-images.githubusercontent.com/2655169/223391219-95117e1e-07a7-43d1-975a-1c96040aa91e.png)

Die Antworten stimmen nicht ganz mit der Realität überein. Ich habe 11 HMIP-SWDO mit einer passenden device_class:

![grafik](https://user-images.githubusercontent.com/2655169/223391888-584d25c8-c58d-4ed9-8b71-d460ad000c0d.png)

Da ich hauptsächlich Homematic Komponenten im Einsatz habe, wollte ich erstmal hier fragen ob schon jemand Erfahrungen gemacht hat. Klappt das bei euch oder muss ich noch ein paar Hausaufgaben machen bevor der Assistent richtig antwortet?

Danke für eure Hilfe!

Beste Grüße
Ingo

**Comment by ChristophCaina** [comment]
Fenster habe ich jetzt im speziellen noch nicht probiert - aber generell meine ich, funktionieren diese Fragen noch nicht so ganz 100%... :)
bei mir aktuell in Bezug auf die Temperaturen in den entsprechenden Räumen... 

Habe es gerade eben mit den Fenstern probiert... gleiches Ergebnis wie bei dir, allerdings glaube ich eher, dass das noch eine Baustelle auf Seiten HomeAssistant und den Intends ist

![image](https://user-images.githubusercontent.com/26391061/223777456-5ee1c654-b19a-4a46-9d09-2a701f318797.png)

**Reply by itn3rd77** [reply]
Hi und danke fürs Testen.

Dann mache ich Mal ein Issue bei Home Assistant auf und schaue was da raus kommt...

**Comment by emufan** [comment]
Hast Du hier schon was rausbekommen? Zur Syntax passt es ja eigentlich:

```wie viele {on_off_domains:domain} (ist|sind) {on_off_states:state} [<area>]```

Bei mir geht aber auch nicht viel. Auch nicht die Abfrage von Temperaturen, usw.

**Reply by itn3rd77** [reply]
Habe den folgenden Issue dazu gefunden: https://github.com/home-assistant/intents/issues/1216

Hört sich nicht nach einer schnellen Lösung an 😖
